packages-exp/auth-exp/src/core/persistence/index.ts

```diff
@@ -31,10 +31,14 @@ export type PersistenceValue = PersistedBlob | string;
 
 export const STORAGE_AVAILABLE_KEY = '__sak';
 
+export type StorageEventListener = (value: PersistenceValue | null) => void;
```

Comment:
Prefer using an interface to describe a function type:
```ts
interface StorageEventListener {
  (value: PersistenceValue|null): void;
}
```

Reply:
done